opentreemap/treemap/lib/map_feature.py

```diff
@@ -315,8 +315,8 @@ def context_dict_for_resource(request, resource, **kwargs):
         __, display_area = get_display_value(instance,
                                              'greenInfrastructure', 'area',
                                              resource.calculate_area())
-        display_units = get_unit_name(get_units(instance,
-                                      'greenInfrastructure', 'area'))
+        display_units = get_unit_abbreviation(
```

Comment:
I am unable to induce a display problem for area using develop. I'm using the chrome window resizer extension and resizing to a width of 768px, "A High-end Windows Phone". I'd say to keep the full-length unit name for area until/unless it becomes a problem.